Library/Homebrew/cask/lib/hbc/cli/internal_stanza.rb

```diff
@@ -68,10 +71,16 @@ def run
             next
           end
 
+          if value.nil? || (value.is_a?(Array) && value.empty?)
```

Comment:
`value.nil? || value&.to_a&.empty?`

Reply:
Unfortunately this errors out:
`Martins-MacBook-Pro:~ martin$ brew cask _stanza version ableton-live`
`Error: undefined method 'to_a' for "9.7.4":Hbc::DSL::Version`
`    Did you mean?  to_yaml`
`             to_r`
`             to_f`
`              to_i`
`              to_s`
`              to_c`

Reply:
Ah, right. Ok, then make the second part `(value.respond_to?(:to_a) && value.to_a.empty?)`.